File: EntityModels/DataTableRequestEM.cs
using System.Collections.Generic;

namespace EntityModels
{
    public class DataTableRequestEM
    {
        public int Start { get; set; }
        public int Length { get; set; }
        public List<ColumnEM> Columns { get; set; }
        public SearchEM Search { get; set; }

        public List<OrderEM> Order { get; set; }
    }

    public class OrderEM
    {
        public int Column { get; set; }
        public bool IsAcs { get; set; }
    }

    public class ColumnEM
    {
        public string Data { get; set; }
        public string Name { get; set; }
        public bool Searchable { get; set; }
        public bool Orderable { get; set; }
        public SearchEM Search { get; set; }
    }

    public class SearchEM
    {
        public string Value { get; set; }
        public string Regex { get; set; }
    }
}

File: EntityModelsVB/DataTableRequestEM.vb
Public Class DataTableRequestEM
    Public Property Start As Integer
    Public Property Length As Integer
    Public Property Columns As List(Of ColumnEM)
    Public Property Search As SearchEM
    Public Property Order As List(Of OrderEM)
End Class

Public Class ColumnEM
    Public Property Data As String
    Public Property Name As String
    Public Property Searchable As Boolean
    Public Property Orderable As Boolean
    Public Property Search As SearchEM
End Class

Public Class SearchEM
    Public Property Value As String
    Public Property Regex As String
End Class


Public Class OrderEM
    Public Property Column As Integer
    Public Property IsAcs As Boolean
End Class